example/src/main/java/io/netty5/example/ocsp/OcspUtils.java

```diff
@@ -150,9 +150,9 @@ public static OCSPResp request(URI uri, OCSPReq request, long timeout, TimeUnit
                     }
 
                     ByteArrayOutputStream baos = new ByteArrayOutputStream();
-                    try {
+                    try (baos) {
```

Comment:
`ByteArrayOutputStream` objects don't actually need to be closed. We can peel off the `try` clause.

Reply:
Let's keep it so it matches our code style.